React-Template-master/src/pages/ForgotPassword.tsx

```diff
@@ -1,18 +1,41 @@
 import { Box, Typography, Grid, TextField, Button } from '@mui/material';
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import LoadingButton from '@mui/lab/LoadingButton';
 import { emailValidation } from '../components/EmailValidation';
+import { auth } from '../components/Firebase';
+import { AuthError, sendPasswordResetEmail } from 'firebase/auth';
+import { getError } from '../components/ErrorHandling';
+import { MessageModal } from '../components/MessageModal';
+
+const MODAL_CONTENT = {
```

Comment:
Why is the name in PascalCase, if you wanted an enum define an enum instead of object

Reply:
@yogendrakvr I have added a type but i didn't change the name style as MODAL_CONTENT is a constant